astropy/utils/tests/test_data.py

```diff
@@ -1140,17 +1140,14 @@ def osraiser(dirnm, linkto, pkgname=None):
 def test_read_unicode(filename):
     contents = get_pkg_data_contents(os.path.join("data", filename), encoding="utf-8")
     assert isinstance(contents, str)
-    contents = contents.splitlines()[1]
-    assert contents == "האסטרונומי פייתון"
+    assert contents.splitlines()[0] == "האסטרונומי פייתון"
 
     contents = get_pkg_data_contents(os.path.join("data", filename), encoding="binary")
     assert isinstance(contents, bytes)
-    x = contents.splitlines()[1]
-    expected = (
-        b"\xff\xd7\x94\xd7\x90\xd7\xa1\xd7\x98\xd7\xa8\xd7\x95\xd7\xa0\xd7\x95"
-        b"\xd7\x9e\xd7\x99 \xd7\xa4\xd7\x99\xd7\x99\xd7\xaa\xd7\x95\xd7\x9f"[1:]
+    assert contents.splitlines()[0] == (
+        b"\xd7\x94\xd7\x90\xd7\xa1\xd7\x98\xd7\xa8\xd7\x95\xd7\xa0\xd7\x95"
+        b"\xd7\x9e\xd7\x99 \xd7\xa4\xd7\x99\xd7\x99\xd7\xaa\xd7\x95\xd7\x9f"
     )
```

Comment:
Do add a comment that `splitlines()` is needed because of Windows. And the following suggestion saves a few characters while making the test stricter.
```suggestion
    assert contents.splitlines() == [
        b"\xd7\x94\xd7\x90\xd7\xa1\xd7\x98\xd7\xa8\xd7\x95\xd7\xa0\xd7\x95"
        b"\xd7\x9e\xd7\x99 \xd7\xa4\xd7\x99\xd7\x99\xd7\xaa\xd7\x95\xd7\x9f"
    ]
```